src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/LogViewer.component.tsx

```diff
@@ -4,6 +4,7 @@ import { FitAddon } from 'xterm-addon-fit';
 import CopyToast, { handleSelectionChange } from './CopyToast';
```

Comment:
Please remove CopyToast if we are not using it

Reply:
We are using it to show popup